pushsafer/pushsafer_test.go

```diff
@@ -0,0 +1,101 @@
+package pushsafer
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/variadico/noti"
+)
+
+func TestEnvConfig(t *testing.T) {
+	cases := []struct {
+		env     noti.MockEnv
+		config  configuration
+		wantErr bool
+		err     error
+	}{
+		{
+			env:     noti.MockEnv{},
+			config:  configuration{},
+			wantErr: true,
+			err:     errNoKey,
+		},
+		{
+			env:     noti.MockEnv{privateKey: "fu"},
+			config:  configuration{},
+			wantErr: false,
+			err:     nil,
+		},
```

Comment:
you changed the wrong line in your last commit. I think this block is supposed to look like:
```go
{
	env:     noti.MockEnv{privateKey: "fu"},
	config:  configuration{},
	wantErr: false,
	err:     nil,
},
```

@variadico does that look right to you?